src/wp-includes/script-loader.php

```diff
@@ -2480,6 +2478,27 @@ function wp_common_block_scripts_and_styles() {
 	do_action( 'enqueue_block_assets' );
 }
 
+/**
+ * Checks if the editor scripts and styles for all registered block types
+ * should be enqueued on the current screen.
+ *
+ * @return boolean
+ */
+function should_load_block_editor_scripts_and_styles() {
```

Comment:
Should this be a part of the public API? I think no. Let's prefix it with `_wp`.

Reply:
I just marked is as private @noisysocks